src/lib/control-appointments.js

```diff
@@ -17,3 +43,91 @@ function setDeleteRowBtn(btn) {
     }
   });
 }
+
+export function addAppointmentRow(appointment) {
+  const { id, creationDate, description, category } = appointment;
+  const row = document.createElement("tr");
+  row.id = id;
+
+  const dateCell = document.createElement("td");
+  dateCell.textContent = creationDate.toDateString();
+
+  const appointmentCell = document.createElement("td");
+  appointmentCell.textContent = description;
+
+  const categoryCell = document.createElement("td");
+  categoryCell.textContent = category;
+
+  const completedCell = document.createElement("td");
+  const completedCheckbox = document.createElement("input");
+  completedCheckbox.type = "checkbox";
+  completedCheckbox.classList.add("checkmark");
+  completedCheckbox.name = "checkbox";
+  completedCheckbox.value = "checkbox";
```

Comment:
this should be replaced by completedCheckbox.checked = !!appointment.completionDate